examples/demo/pom.xml

```diff
@@ -238,12 +243,12 @@
 			</activation>
 			<modules>
 				<module>domain</module>
-				<module>javafx</module>
+				<!--module>javafx</module-->
```

Comment:
any reason why this was removed?

Reply:
couldn't build locally - committed by accident